packages/devtools_app/lib/src/primitives/enum_utils.dart

```diff
@@ -29,7 +27,7 @@ class EnumUtils<T> {
   final Map<String, T> _lookupTable = {};
   final Map<T, String> _reverseLookupTable = {};
 
-  T enumEntry(String enumName) => _lookupTable[enumName];
+  T? enumEntry(String? enumName) => _lookupTable[enumName!];
```

Comment:
if we are going to always force unwrap `enumName` maybe we shouldn't allow the parameter to be nullable